app/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,177 +1,247 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/activity_main"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/main_background"
-    android:orientation="vertical">
-
-    <TextView
-        android:id="@+id/calculation_label"
+    android:layout_width="fill_parent"
```

Comment:
Maros, this state of the property now is deprecated, so you need to change to match or wrap content.

Reply:
Done, Maro thanks